docs/terms/expectation_suite.md

```diff
@@ -3,12 +3,83 @@ id: expectation_suite
+
+Expectation Suites are stored in an <TechnicalTag relative="../" tag="expectation_store" text="Expectation Store" />.  They are generated by <TechnicalTag relative="../" tag="profiler" text="Profilers" />, and are used by <TechnicalTag relative="../" tag="checkpoint" text="Checkpoints" /> to <TechnicalTag relative="../" tag="validation" text="Validate" /> data.
+
+## Use cases
+
+<CreateHeader/>
+
+The lifecycle of an Expectation Suite starts with creating it. Then it goes through an iterative loop of Review and Edit as the team's understanding of the data described by the suite evolves.
+
+Expectation Suites are largely managed automatically in the workflows for creating Expectations.  When the Expectations are created, an Expectation Suite is created to contain them.  In the Profiling workflow, this Expectation Suite will contain all the Expectations generated by the Profiler.  In the interactive workflow, an Expectation Suite will be configured to include Expectations as they are defined, but will not be saved to an Expectation Store until you issue the command for it to be.
+
+For more information on these processes, please see:
+- [Our overview on the process of Creating Expectations](../guides/expectations/create_expectations_overview.md)
+- [Our guide on how to create and edit Expectations with a Profiler](../guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md)
+- [Our guide on how to create and edit Expectations with instant feedback from a sample Batch of data](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md)
+
+<ValidateHeader/>
+
+Expectation Suites are *used* during the Validation of data.  In this step, you will need to provide one or more Expectation Suites to a Checkpoint.  This can either be done by configuring the Checkpoint to use a preset list of one or more Expectation Suites, or by configuring the Checkpoint to accept a list of one or more Expectation Suites at runtime.
+
+## Features
+
+### CRUD operations
+
+A Great Expectations Expectation Suite enables you to perform Create, Read, Update, and Delete (CRUD) operations on the Suite's Expectations without needing to re-run them. 
+
+### Reusability
+
+Expectation Suites are primarily used by Checkpoints, which can accept a list of one or more Expectation Suite.  Because they are stored independently of the Checkpoints that use them, the same Expectation Suite can be included in the list for multiple Checkpoints, provided the Expectation Suite contains a list of Expectations that describe the data that Checkpoint will Validate.  This allows you to, for example, create multiple Checkpoints with increasing vigor of in their validation, or create multiple Checkpoints with the same base set of Expectations and some additional Expectations tailored to specific data (such as one Checkpoint for validating raw data, and one with additional Expectation Suites for validating post-processing data).
```

Comment:
This section is confusing to me. I understand the notion that the same suite can be applied to multiple datasets, but I'm not sure where we're trying to go with the discussion of increasing rigor. Can you clarify?

Reply:
I'm basing this on my experience working in GIS.  We would often be given raw data that would then need to be formatted using automated processes before being passed on to technicians to have additional details filled in and records verified against alternate datasources.  Ultimately, we would end up with at least three related levels of validation being needed, covering data that had increasing rigidity of its standardization -- raw data, pre-(manual) processing, and post (manual) processing.  The most sensible way of doing this in GE, I thought, would be to start with one Expectation Suite and Checkpoint for the raw data, to detect changes that may have occurred there.  A second Checkpoint would include that first Expectation Suite and an additional Expectation Suite to validate that none of the automated steps in generating the pre-(manual) processing data had behaved unexpectedly, as can happen when scripts are being refined (as would be required if the first Checkpoint flagged a change in how the raw data was formatted) while production is underway.  And then a third Checkpoint which combined the first two Expectation Suites and a third Expectation Suite asserting that all required fields were populated, etc, would be applied as quality control to the post-process data.  With each step in the cleaning, clarifying, and expanding of the data you would ensure that the previous Expectations hadn't been violated as a sort of quality control check, while also verifying that all of the additional Expectations you would have for data that had gone through an automated cleanup and then through technician processing were also being met, without having to rewrite the Expectations from the earlier iterations of the data.

Reply:
Discussed life; verifying the behavior here, but we think it's the case that the list of validations is a list of (batch_request, suite) pairs.

Reply:
Verified: Checkpoints accept one or more Execution Suite and Batch Request pairs.  I've updated this section accordingly.